manifests/init.pp

```diff
@@ -27,18 +27,26 @@
 #
 # @param manage_properties
 #   Boolean specifying whether or not to manage the catalina.properties file. Defaults to true.
+#
+# [*copy_from_home*]
```

Comment:
The inline docs have been updated for eventually being used by puppet-strings documentation generation. You can see the format above.